shared-ui/style/typography.ts

```diff
@@ -50,7 +50,6 @@ const P = styled.p`
   margin: 0;
   font-family: ${fonts.nunitoSansRegular};
   font-size: 1.1em;
-  line-height: 1.4em;
```

Comment:
Don't remove this line height here since it's shared across all the P's in the website and this line height was created according to the design for P's. Instead, add to your styled component what line-height you'd like your styled component to be instead to override this